### PR TITLE
fix(core): remove joined workspace icon from current worksapce card

### PR DESCRIPTION
**Pull request by JimmFly**
close PD-1669

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#8117** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8117?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/components/workspace-selector/workspace-card/index.tsx [hunk 1]
```diff
@@ -314,7 +316,9 @@ export const WorkspaceCard = forwardRef<
             Enable Cloud
           </Button>
         ) : null}
-        {information?.isOwner ? null : <CollaborationIcon />}
+        {disableCollaborationIcon || information?.isOwner ? null : (
```
Comment (pengx17) [inline packages/frontend/core/src/components/workspace-selector/workspace-card/index.tsx]: Just remove it or show on hovering?

Comment (JimmFly) [reply packages/frontend/core/src/components/workspace-selector/workspace-card/index.tsx]: Both menu trigger and menu item use the workspace card, so we cannot remove it directly. We need to display this icon in the workspace list.

Comment (pengx17) [reply packages/frontend/core/src/components/workspace-selector/workspace-card/index.tsx]: I see. but the option might be renamed to hide xxx icon

**Review by pengx17 — APPROVED** [review]

**Pull request merged**